diff --git a/gui/electron/main/index.ts b/gui/electron/main/index.ts
index 3f1f1dfd..404e22c9 100644
--- a/gui/electron/main/index.ts
+++ b/gui/electron/main/index.ts
@@ -386,6 +386,8 @@ const checkEnvironmentVariables = () => {
 const isServerRunning = async () => !(await isPortAvailable(21110));
 
 const spawnServer = async () => {
+  return;
+
   if (options.skipServerIfRunning && (await isServerRunning())) {
     logger.info(
       { skipServerIfRunning: options.skipServerIfRunning },
