diff --git a/wayvr/src/state.rs b/wayvr/src/state.rs
index fb2363f..ba74300 100644
--- a/wayvr/src/state.rs
+++ b/wayvr/src/state.rs
@@ -168,7 +168,7 @@ impl AppState {
         let mut theme = WguiTheme {
             animation_mult: 1. / session.config.ui_animation_speed,
             rounding_mult: session.config.ui_round_multiplier,
-            ..Default::default()
+            gradient_intensity: session.config.ui_gradient_intensity,
         };
 
         let dbus = DbusConnector::default();
